cpp/test/Slice/errorDetection/NestedModule.ice

```diff
@@ -5,7 +5,6 @@ module All::Good::Here
 {
```

Comment:
Same here, it ate lines.
Indeed, seems to be something about single-line definitions.